packages/playback/src/lib/consts/text-tracks.ts

```diff
@@ -0,0 +1,17 @@
+export enum TextTrackKind {
+  Subtitles = 'subtitles',
+  Captions = 'captions',
+  Descriptions = 'descriptions',
+  Chapters = 'chapters',
+  Metadata = 'metadata',
+  // webkit only
+  Forced = 'forced',
+}
+
+export enum TextTrackMode {
+  Disabled = 'disabled',
+  Hidden = 'hidden',
+  Showing = 'showing',
+}
```

Comment:
these 2 seem to match default text track interfaces, so you can simply use interfaces globally available in typescript via dom